src/ocean/time/Time_test.d

```diff
@@ -51,3 +53,16 @@ unittest
     test(tod.seconds is 3);
     test(tod.millis is 4);
 }
+
+// pretty time formatting
+unittest
+{
+    test!("==")(
+        format("{}", asPrettyStr(Time.epoch1970)),
+        "01/01/70 00:00:00"
```

Comment:
I sure hope the locale on the test machines will not change :-)

Reply:
A _lot_ of tests will break when that happens :)